emba

```diff
@@ -368,6 +368,28 @@ main() {
   emba_parameter_parsing "$@"
   set_log_paths
 
+  # Validate RESCAN_SBOM mode requirements
+  if [[ "${RESCAN_SBOM}" -eq 1 ]]; then
+    if [[ -z "${LOG_DIR}" ]] || [[ "${LOG_DIR}" == *"/logs" ]]; then
```

Comment:
I think this check is not fully working as expected. Why not checking for the SBOM file itself?

```
    if ! [[ -f "${LOG_DIR}/SBOM/EMBA_cyclonedx_sbom.json" ]]; then
```

Or, better way would be to define the SBOM path in the defaults and reuse it everywhere